src/domains/printit-app/01_app_service_pdf_engine_ha.tf

```diff
@@ -1,460 +1,460 @@
-#         metric_resource_id       = module.printit_pdf_engine_app_service_java_ha[0].id
-#         metric_namespace         = "microsoft.web/sites"
-#         time_grain               = "PT1M"
-#         statistic                = "Average"
-#         time_window              = "PT5M"
-#         time_aggregation         = "Average"
-#         operator                 = "LessThan"
-#         threshold                = 2 #sec
-#         divide_by_instance_count = false
-#       }
-#
-#       scale_action {
-#         direction = "Decrease"
-#         type      = "ChangeCount"
-#         value     = "1"
-#         cooldown  = "PT20M"
-#       }
-#     }
-#
-#     # CpuPercentage
-#
-#     # Supported metrics for Microsoft.Web/sites
-#     # 👀 https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-web-sites-metrics
-#     rule {
-#       metric_trigger {
-#         metric_name              = "CpuPercentage"
-#         metric_resource_id       = module.printit_pdf_engine_app_service_java_ha[0].plan_id
-#         metric_namespace         = "microsoft.web/serverfarms"
-#         time_grain               = "PT1M"
-#         statistic                = "Average"
-#         time_window              = "PT5M"
-#         time_aggregation         = "Average"
-#         operator                 = "GreaterThan"
-#         threshold                = 75
-#         divide_by_instance_count = false
-#       }
-#
-#       scale_action {
-#         direction = "Increase"
-#         type      = "ChangeCount"
-#         value     = "2"
-#         cooldown  = "PT5M"
-#       }
-#     }
-#
-#     rule {
-#       metric_trigger {
-#         metric_name              = "CpuPercentage"
-#         metric_resource_id       = module.printit_pdf_engine_app_service_java_ha[0].plan_id
-#         metric_namespace         = "microsoft.web/serverfarms"
-#         time_grain               = "PT1M"
-#         statistic                = "Average"
-#         time_window              = "PT5M"
-#         time_aggregation         = "Average"
-#         operator                 = "LessThan"
-#         threshold                = 30 #sec
-#         divide_by_instance_count = false
-#       }
-#
-#       scale_action {
-#         direction = "Decrease"
-#         type      = "ChangeCount"
-#         value     = "1"
-#         cooldown  = "PT20M"
-#       }
-#     }
-#   }
-# }
-#
+################
+# node
+################
+
+
+module "printit_pdf_engine_app_service_ha" {
+  source              = "git::https://github.com/pagopa/terraform-azurerm-v3.git//app_service?ref=v8.5.0"
+  count               = var.is_feature_enabled.pdf_engine_ha ? 1 : 0
+  vnet_integration    = false
+  resource_group_name = azurerm_resource_group.printit_pdf_engine_app_service_rg.name
+  location            = var.location
+
+  # App service plan vars
+  plan_name = format("%s-plan-pdf-engine-ha", local.project)
+  sku_name  = var.app_service_pdf_engine_sku_name
```

Comment:
@diegolagospagopa @mamari90 we need reservations for `P2v3` app services sku type used by node PDF engine printit and receipt 

cc @jacopocarlini for printit, unlike receipts evaluate at the beginning in prod to start with a default of 1 and not 3 instances 

I think it will not be an intensely used service

Reply:
I used `S1` for DEV at the moment